packages/playground-ui/src/ds/components/Drawer/drawer.css

```
@@ -0,0 +1,130 @@
+}
+
+/* Release after a swipe: scale the settle duration by the gesture velocity. */
+.drawer-popup[data-ending-style] {
+  transition-duration: calc(var(--drawer-swipe-strength, 1) * 400ms);
+}
+
+/* ---- Bottom sheet (swipe down) ---------------------------------------------- */
+
+.drawer-popup[data-swipe-direction='down'] {
+  --live-y: calc(var(--drawer-snap-point-offset, 0px) + var(--drawer-swipe-movement-y, 0px));
+  transform-origin: 50% calc(100% - var(--bleed));
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)